ServiceRepositoryPattern/ServiceRepositoryPattern/Repository/FoodRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ServiceRepositoryPattern.Models;
+
+namespace ServiceRepositoryPattern.Repository
+{
+    public class FoodRepository : IFoodRepository
+    {
+        public List<FoodItem> GetAllSold()
+        {
+            List<FoodItem> items = new List<FoodItem>()
+            {
+                new FoodItem()

```

Comment:
why these hardcoded values....you were supposed to create data using API right?

Reply:
I have created API for FoodItem CRUD these are just for practice.